templates/Library/NuGet/.template.config/template.json

```diff
@@ -40,6 +40,30 @@
       "type": "parameter",
       "dataType":"bool",
       "defaultValue": "false"
+    },
+    "ci-provider": {
```

Comment:
name this `pipeline` rather than `ci-provider`.  Update the readme to reflect this change.